backend/routes/Instructors.go

```diff
@@ -0,0 +1,32 @@
+package routes
+
+import (
+	"net/http"
+
+	"github.com/ucsdscheduleplanner/UCSD-Schedule-Planner/backend/db"
+)
+
+const logTagInstructors = "[Instructors]"
+
+// GetInstructors is a pre-http.HandlerFunc for instructor route and will become a closure with *DatabaseStruct
+func GetInstructors(writer http.ResponseWriter, request *http.Request, ds *db.DatabaseStruct) {
+
+	if request.Method != "GET" {
+		errInvalidMethod(logTagInstructors, writer, request)
+		return
+	}
+
+	department, courseNum, quarter, missing := readURLQueryDeptCourseNumQuarter(request)
```

Comment:
you make your method names too long in an attempt to make them descriptive. I would do something like GetFromURL(), short and simple and people can understand as they read over it